en/micro-integrator/docs/use-cases/examples/rabbitmq_examples/retry-delay-failed-msgs-rabbitmq.md

```diff
@@ -1,7 +1,12 @@
 # Control the number of retries and delay message in case of error
 
-!!! Note
-    <b>Work in progress!</b>
+This sample demonstrates how you can implement a scenario to add delays and control the number of retries in
```

Comment:
what are we delaying and what are we retrying?
Is this consuming RabbitMQ messages or producing RabbitMQ messages?

Reply:
we are retrying to send the message again, upon a delay.
If the consumer proxy fails to deliver the message the first time, the message will be moved to a dead-letter-exchange. We have to configure the RabbitMQ to move this message back to the original queue (the one which the consumer proxy is listening) upon a delay.